packages/mongodb-chatbot-server/src/routes/responses/createResponse.test.ts

```diff
@@ -437,7 +455,11 @@ describe("POST /responses", () => {
         });
 
       expect(response.statusCode).toBe(400);
-      expect(response.body).toEqual({ error: "Invalid request" });
+      expect(response.body.error).toEqual(
+        badRequestError(
+          "Path: body.metadata.key1 - String must contain at most 512 character(s)"
```

Comment:
```suggestion
          "Path: body.metadata.key1 - String must contain at most 512 characters"
```

nit

Reply:
This is actually generated by zod like that, would have to override with a custom refine to drop the parens 😓